graph/ui-project-details/src/lib/target-configuration-details/target-configuration-details.tsx

```diff
@@ -109,7 +111,16 @@ export default function TargetConfigurationDetails({
               />
             </h4>
             <p className="pl-5 font-mono">
-              <TargetExecutor {...displayHeader} link={link} />
+              <TargetExecutor {...displayHeader} link={link}>
+                {targetSourceInfo && (
+                  <span className="pl-4 opacity-0 transition-opacity duration-150 ease-in-out group-hover/line:opacity-100">
+                    <SourceInfo
+                      data={targetSourceInfo}
+                      propertyKey={`targets.${targetName}`}
```

Comment:
This source information is wrong. It shoudl be the source information of the actual command/script/executor.